declarations/Facebook.json

```
@@ -55,11 +55,11 @@
     "Data Processor Agreement": {
       "fetch": "https://www.facebook.com/legal/EU_data_transfer_addendum",
       "select": [
-        ".ad2k81qe"
+        "div[role=main"
```

Your selector works but it is not closed properly.

```suggestion
"div[role=main]"
```